MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -66,9 +66,31 @@ public MediaInstance(Guid MediaAssetId)
 		private ActorTransformPatch _rbTransformPatch;
 
 		private new Renderer renderer = null;
-		internal Renderer Renderer => renderer = renderer ?? GetComponent<Renderer>();
-		private MeshFilter filter = null;
-		internal MeshFilter MeshFilter => filter = filter ?? GetComponent<MeshFilter>();
+		internal Renderer Renderer
+		{
+			get
+			{
+				if (renderer == null)
+				{
+					var t = GetComponent<Renderer>();
+					renderer = t;
```

Comment:
Why have the temp variable? Here and below.

Reply:
Oh that was for debugging. Will make a note to remove.